WORKSPACE

```diff
@@ -199,6 +199,30 @@ http_archive(
 load("@pybind11_bazel//:python_configure.bzl", "python_configure")
 python_configure(name = "local_config_python")
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```

Comment:
Why do we need re-define http_archive? We are already using it above multiple times (line 193 for example)